### PR TITLE
Removing URL when unpublished

### PR DESCRIPTION
**Pull request by mviterna**
## Original issue(s)
https://vajira.max.gov/browse/API-6703

## Things to know about this PR

No to all

Local testing

### CODE REVIEW

File: modules/va_forms/app/workers/va_forms/form_reloader.rb [hunk 1]
```diff
@@ -87,7 +87,11 @@ def init_attributes(form)
         benefit_categories: map_benefit_categories(form['fieldBenefitCategories']),
         va_form_administration: form.dig('fieldVaFormAdministration', 'entity', 'entityLabel')
       }
-      mapped[:form_details_url] = "#{FORM_BASE_URL}#{form.dig('entityUrl', 'path')}" if form['entityPublished']
+      mapped[:form_details_url] =
+        if form['entityPublished']
+          "#{FORM_BASE_URL}#{form.dig('entityUrl', 'path')}"
+        else
+          '' end
```
Comment (rileyanderson) [inline modules/va_forms/app/workers/va_forms/form_reloader.rb:94]: Just a style change here, the `end` should be on the next line 

```ruby
if form['entityPublished']
  "#{FORM_BASE_URL}#{form.dig('entityUrl', 'path')}"
else
  ''
end
```

**Review by rileyanderson — CHANGES_REQUESTED** [review]

File: modules/va_forms/app/workers/va_forms/form_reloader.rb (hunk 1 above)
Comment (mviterna) [reply modules/va_forms/app/workers/va_forms/form_reloader.rb]: I was thinking Rubocop forced me to do it that way, but I could have dreamed that up. Let me try it again...

Comment (mviterna) [reply modules/va_forms/app/workers/va_forms/form_reloader.rb]: @rileyanderson Yeah, if I add a newline, rubocop says the method is too long. Do you have a preference how you want to get past that? If I put it all on one line, it looks too long to me

Comment (rileyanderson) [reply modules/va_forms/app/workers/va_forms/form_reloader.rb]: Ah yeah it looks like rubocop is catching too many lines in the method 😞 . You could use a ternary operator here to keep method lines down and to keep styling consistent 

```ruby
mapped[:form_details_url] = form['entityPublished'] ? "#{FORM_BASE_URL}#{form.dig('entityUrl', 'path')}" : ''
```

Comment (mviterna) [reply modules/va_forms/app/workers/va_forms/form_reloader.rb]: done

**Review by rileyanderson — APPROVED** [review]
LGTM

**Pull request merged**